src/components/Button.js

```diff
@@ -1,14 +1,37 @@
 import React from 'react';
-import propTypes from 'prop-types';
+import PropTypes from 'prop-types';
 
-const Button = ({ name, onClick }) => (
-  <button onClick={onClick} type="button">
-    {name}
-  </button>
-);
+const Button = props => {
+  const {
+    onClick, color, wide, name,
+  } = props;
+
+  return (
+
+    <button
+      className="bold-size "
+      onClick={onClick}
+      type="button"
+      color={color}
```

Comment:
color = {color} is not needed in here. So please remove it.

Reply:
Thank you for your quick and effective review. I will remove it :+1: